ha/core/config/config_manager.py

```diff
@@ -77,6 +77,27 @@ def load_controller_schema():
         """
         ConfigManager._safe_load(const.CM_CONTROLLER_INDEX, f"json://{const.CM_CONTROLLER_SCHEMA}")
 
+    @staticmethod
+    def load_filter_rules():
+        """
+        Loads alert filter rules.
+        """
+        ConfigManager._safe_load(const.ALERT_FILTER_INDEX, f"json://{const.ALERT_FILTER_RULES_FILE}")
+
+    @staticmethod
+    def get_filter_type():
```

Comment:
get_filter_type
get_resource_types_list

It is not needed

directly get from filter as they not required by other component
config_manager.py is good for global level configuration like version

Reply:
Agree. Addressed comment